### PR TITLE
sound: Prepare v0.2.0 release

### PR DESCRIPTION
**Pull request by MatiasVara**
### Summary of the PR

Update changelog and Cargo.toml to v0.2.0

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR have Signed-Off-By trailers (with
  `git commit -s`), and the commit message has max 60 characters for the
  summary and max 75 characters for each description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] All added/changed public-facing functionality has entries in the "Upcoming 
  Release" section of CHANGELOG.md (if no such section exists, please create one).
- [x] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

File: vhost-device-sound/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,20 @@
...
+
+### Changed
+- Update pipewire dependencies to version 0.8 after the release of pipewire
+  v0.8
+
+### Fixed
+- Fix symbolic links to license files
+- Remove duplicate increment in pipewire
+- Destroy pipewire streams not destroyed in pipewire
+
```
Comment (stefano-garzarella) [inline vhost-device-sound/CHANGELOG.md:25]: What about adding links to the PRs, something like we do in vhost: https://raw.githubusercontent.com/rust-vmm/vhost/main/vhost/CHANGELOG.md

File: vhost-device-sound/CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,20 @@
 
 ### Deprecated
 
+## v0.1.1
```
Comment (stefano-garzarella) [inline vhost-device-sound/CHANGELOG.md:12]: I was thinking whether we should release v0.2.0, but maybe we haven't changed anything from the user's point of view, so I think v.0.1.1 is fine. Opinions?

**Review by stefano-garzarella — COMMENTED** [review]
Great, thanks for working on this. I'm fine with a new release, I just left little comments.

**Comment by stefano-garzarella** [comment]
I think we should also commit `Cargo.lock` changes. After the build I have this diff:

```diff
diff --git a/Cargo.lock b/Cargo.lock
index 5bce7e3..6bab63f 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -1481,7 +1481,7 @@ dependencies = [
 
 [[package]]
 name = "vhost-device-sound"
-version = "0.1.0"
+version = "0.1.1"
 dependencies = [
  "alsa",
  "clap",
```

**Comment by stefano-garzarella** [comment]
I'm thinking if we should include also #580 in this release.

File: vhost-device-sound/CHANGELOG.md (hunk 2 above)
Comment (epilys) [reply vhost-device-sound/CHANGELOG.md:12]: The only user facing change here seems to be the `pipewire` dependency; does it require updated system dependencies for the end-user? If yes, a `0.2.0` would make sense I think.

**Review by stefano-garzarella — APPROVED** [review]

**Review by epilys — APPROVED** [review]

**Pull request merged**